server/views/layouts/main.hbs

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
Please remember to always [set the language](https://www.w3.org/International/questions/qa-html-language-declarations) of your doc (i.e. `en-gb`) for assistive tech.

Reply:
Fixed 👍